webnn/conformance_tests/elementwise_logical.https.any.js

```diff
@@ -7,14 +7,19 @@
 
 // https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-logical
 
-testWebNNOperation(
-  [
-    'equal',
-    'greater',
-    'greaterOrEqual',
-    'lesser',
-    'lesserOrEqual',
-  ],
-  buildOperationWithTwoInputs
-);
-testWebNNOperation('logicalNot', buildOperationWithSingleInput);
\ No newline at end of file
+if (navigator.ml) {
```

Comment:
Here's modification in `elementwise_logical.https.any.js` test file. @jgraham PTAL, thanks.